### PR TITLE
some small changes

### PR DESCRIPTION
**Pull request by iroi44**
@siddontang @qiuyesuifeng PTAL

### CODE REVIEW

File: op-guide/monitor-overview.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 TiDB 使用开源时序数据库 Prometheus 作为监控和性能指标信息存储方案，使用 Grafana 作为可视化组件进行展示。
 
-Prometheus 的是一个拥有多纬度数据模型，灵活的查询语句的时序数据库。作为热门的开源项目之一的它，拥有活跃的社区，及优秀的案例。
-
-同时作为 Prometheus 社区的活跃开发者的 PingCAP，以及对集群的监控需求，我们采用 Prometheus 来对TiDB/TiKV/PD 来进行监控和报警，并应用在我们的系统中。
+Prometheus 是一个拥有多纬度数据模型，灵活的查询语句的时序数据库。Prometheus 作为热门的开源项目，拥有活跃的社区及众多的成功案例。
```
Comment (overvenus) [inline op-guide/monitor-overview.md]: 是`多维度`，还是`多纬度`？

Comment (c4pt0r) [inline op-guide/monitor-overview.md]: '维度' is correct.

**Comment by siddontang** [comment]
LGTM

**Comment by iroi44** [comment]
改了，PTAL @siddontang @c4pt0r @shenli

**Comment by c4pt0r** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**